### ISSUE TITLE
Data Collection presentation is very SmarGon-centric

### ISSUE THREAD

**Issue by dsclassen** [DiamondLightSource/SynchWeb#100]
I understand that the main air-bearing axis of the SmarGon goniometer is labelled as Omega and therefore is the primary axis for data collection, but this isn't necessarily the case at other beamlines.

The DataCollection table in ISPyB has a column rotationAxis. I assumed this column should indicate the axis of rotation used during the data collection. Even if I have "Phi" in this column SyncWeb shows my rotation axis as Omega.

Judicious grepping has revealed that SynchWeb has hardcoded Omega (in the form of `&Omega;`) for various Data Collection parameters:

I can of course just do a global search and replace to make things look proper for our beamline, but maybe there is a better way to do this? Extract it to the config file? 

I will see if I can figure out something and maybe even contribute a PR, but I must admit my javascript/marionette/webby skills are...under development.

```
SynchWeb/client/src/js/templates/imaging/queuexfe.html:		<td class="renderable">&Omega; Range</td>
SynchWeb/client/src/js/templates/imaging/ssdiffractionplan.html:            <span class="label">Omega Start</span>
SynchWeb/client/src/js/templates/imaging/ssdiffractionplan.html:            <span class="label">Omega Range</span>
SynchWeb/client/src/js/templates/imaging/blparamlimitsstatic.html:        <td class="renderable">&Omega;: </td>
SynchWeb/client/src/js/templates/imaging/queuegrid.html:		<td class="renderable">&Omega; Range</td>
SynchWeb/client/src/js/templates/imaging/blparamlimits.html:        <td class="renderable">&Omega;: </td>
SynchWeb/client/src/js/templates/imaging/queuepoint.html:		<td class="renderable">&Omega; Start</td>
SynchWeb/client/src/js/templates/imaging/queuepoint.html:		<td class="renderable">&Omega; Range</td>
SynchWeb/client/src/js/templates/imaging/queuepoint.html:		<td class="renderable">&Omega; End</td>
SynchWeb/client/src/js/templates/imaging/savepreset.html:            <span class="label">Omega Start</span>
SynchWeb/client/src/js/templates/imaging/savepreset.html:            <span class="label">Omega Range</span>
SynchWeb/client/src/js/templates/cell/item.html:            <li>&Omega; Start: <%-AXISSTART%>&deg;</li>
SynchWeb/client/src/js/templates/cell/item.html:            <li>&Omega; Osc: <%-AXISRANGE%>&deg;</li>
SynchWeb/client/src/js/templates/dc/grid.html:        <li>&Omega; Start: <%-AXISSTART%>&deg;</li>
SynchWeb/client/src/js/templates/dc/reprocess_dc.html:    <li>&Omega; Start: <%-AXISSTART%>&deg;, Osc: <%-AXISRANGE%>&deg;</li>
SynchWeb/client/src/js/templates/dc/dc.html:        <li>&Omega; Start: <%-AXISSTART%>&deg;</li>
SynchWeb/client/src/js/templates/dc/dc.html:        <li>&Omega; Osc: <%-AXISRANGE%>&deg;</li>
SynchWeb/client/src/js/templates/dc/dc.html:        <li>&Omega; Overlap: <%-OVERLAP%>&deg;</li>
SynchWeb/client/src/js/templates/dc/dc_strategy.html:            <th>&Omega; Start</th>
SynchWeb/client/src/js/templates/dc/dc_strategy.html:            <th>&Omega; Osc</th>
SynchWeb/client/src/js/templates/mc/datacollection.html:<span>&Omega; Start: <%-AXISSTART%>&deg; &Omega; Osc: <%-AXISRANGE%>&deg; | <a title="Click to view diffraction images for this data collection" class="images" href="/dc/view/id/<%-ID%>">Images</a></span>
```

**Comment by stufisher** [comment]
I've done a bit of work on this, ill add a pr shortly

**Comment by stufisher** [comment]
Rough pr here: https://github.com/DiamondLightSource/SynchWeb/pull/103
needs some testing

**Comment by stufisher** [comment]
Resolved in #103 which is merged
